<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDiagramDialogBase</class>
 <widget class="QDialog" name="QgsDiagramDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="mDisplayDiagramsCheckBox">
     <property name="text">
      <string>Display diagrams</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="mTypeLabel">
       <property name="text">
        <string>Diagram type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mDiagramTypeComboBox"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QStackedWidget" name="mDiagramFactoryStackedWidget">
     <widget class="QWidget" name="page_3"/>
     <widget class="QWidget" name="page_4"/>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="1">
      <widget class="QComboBox" name="mClassificationTypeComboBox"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QStackedWidget" name="mWidgetStackRenderers">
       <widget class="QWidget" name="page"/>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mClassificationLabel">
       <property name="text">
        <string>Classification attribute</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="mClassificationComboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mClassificationTypeLabel">
       <property name="text">
        <string>Classification type</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
